openlibrary/templates/login.html

```diff
@@ -12,7 +12,7 @@ <h1>$_("Log In")</h1>
 
   $if ctx.user:
       <p>$:_("You are already logged into Open Library as %(user)s.", user='<a href="%s">%s</a>' % (ctx.user.key, ctx.user.displayname))</p>
-      <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[\'logout\'].submit()">log out</a> and start the signup process afresh.')</p>
+      <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[\'logout\'].trigger('submit')">log out</a> and start the signup process afresh.')</p>
```

Comment:
```suggestion
      <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[\'logout\'].trigger(\'submit\')">log out</a> and start the signup process afresh.')</p>
```

If single quotes surrounding `submit` are not escaped, `login.html` throws an error.

Reply:
Disregard previous suggestion.  This should be rolled back to the original code, as `document.forms.logout` is not a jQuery object.